sites/charmhub.io.yaml

```diff
@@ -10,6 +10,9 @@ env:
   - name: CHARMSTORE_API_URL
     value: https://api.staging.snapcraft.io/
 
+  - name: LOGIN_GROUP
```

Comment:
could you be a little bit more specific on those environment variable.

`LOGIN_LAUNCHPAD_TEAM`

Reply:
Only if you review this PR ;P
https://github.com/canonical-web-and-design/charmhub.io/pull/171